src/main/gov/nasa/jpf/listener/VarTracker.java

```diff
@@ -191,7 +191,9 @@ public void instructionExecuted(VM vm, ThreadInfo ti, Instruction nextInsn, Inst
       varId = ((JVMFieldInstruction)executedInsn).getFieldName();
 
       StackFrame frame = ti.getModifiableTopFrame();
-      frame.addOperandAttr(varId);
+      if (frame.getTopPos() >= 0) {
```

Comment:
I think this check is also needed a few lines above (after line 185), as I don't think we can be sure that there is a local variable in all cases where that line is hit.
WDYT?